example-get-started-experiments/code/.github/workflows/run-experiment.yml

```diff
@@ -64,32 +65,27 @@ jobs:
           STUDIO_TOKEN: ${{ secrets.STUDIO_TOKEN }}
           DVCLIVE_LOGLEVEL: DEBUG
         run: |
-          dvc pull
           cml ci --fetch-depth 0
+          dvc pull
```

Comment:
Any reason for this instead of `dvc exp run --pull --allow-missing`?

Reply:
The fix was not released at the point I used this. Will update